Sources/MountebankSwift/Models/Imposter/Imposter.swift

```diff
@@ -37,20 +77,10 @@ public struct Imposter: Codable, Equatable {
     /// By retrieving the imposter, your client code can determine if an expected service call was in fact made.
     public let requests: [Imposter.RecordedRequest]?
 
-    enum CodingKeys: String, CodingKey {
-        case port
-        case networkProtocol = "protocol"
-        case name
-        case stubs
-        case recordRequests
-        case defaultResponse
-        case numberOfRequests
-        case requests
-    }
-
     public init(
         port: Int? = nil,
         networkProtocol: NetworkProtocol,
+        extraNetworkOptions: ExtraNetworkOptions? = nil,
```

Comment:
Why not put these options directly in the `NetworkProtocol` as associated values?

Reply:
I did not change it because the NetworkProtocol was also used in the `Imposters.swift`. But now created just one for that one.